lab_4_fill_words_by_ngrams/start.py

```diff
@@ -10,7 +15,35 @@ def main() -> None:
     """
     with open("./assets/Harry_Potter.txt", "r", encoding="utf-8") as text_file:
         text = text_file.read()
-    result = None
+    processor = WordProcessor('<eow>')
+    encoded = processor.encode(text)
+    model = NGramLanguageModel(encoded, 2)
+    model.build()
+    generator = TopPGenerator(model, processor, 0.5)
+    result = generator.run(51, 'Vernon')
+    print(result, '\n')
+
+    gen_types = GeneratorTypes()
+    gens = {gen_types.greedy: GreedyTextGenerator(model, processor),
```

Comment:
why do not use get_conversion_generator_type ?

Reply:
get_conversion_generator_type returns a string with name, but for the QualityChecker we need a dictionary of a numeric type and generator, so using it would not help at all :(